discord/invite.py

```diff
@@ -362,6 +369,14 @@ def __repr__(self):
     def __hash__(self):
         return hash(self.code)
 
+    @property
+    def type(self):
+        """:class:`InviteUserTarget`: The invite's Discord type.
```

Comment:
```suggestion
        """:class:`InviteUserTarget`: The invite's :attr:`target_user` type.
```
And maybe reference the Discord API docs for clarity?